money_bot/utils/db_utils.py

```diff
@@ -17,8 +21,13 @@
     await user.commit()
 
 
-async def get_user_by_id(user_id: int):
-    return await models.User.find_one({"user_id": user_id})
+async def get_user_money_amount(user_id: int):
```

Comment:
Pls use this function also in balance module

Reply:
done